elasticdl_preprocessing/tests/to_number_test.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+from elasticdl_preprocessing.layers.to_number import ToNumber
+
+
+def sparse_tensor_equal(sp_a, sp_b):
```

Comment:
We may use the method in multiple unit tests of preprocessing layers. Can we move it to a single file like `tests/test_utils.py` ?

Reply:
Good idea.